.github/workflows/customer-service.yml

```diff
@@ -0,0 +1,27 @@
+name: customer-service-pipeline
+on:
```

Comment:
Since the setting for push and pull_requests is the same, did you consider to use
`# Triggers the workflow on push or pull request events
on: [push, pull_request]`?

Reply:
I think it is not possible if I understand the doc correctly `If you need to specify activity types or configuration for an event, you must configure each event separately. You must append a colon (:) to all events, including events without configuration.`
see: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions